app/controllers/users_controller.rb

```diff
@@ -38,14 +38,10 @@ def update
   end
 
   def update_privacy_settings
-    privacy_params = params.fetch(:user).permit(:strip_exif)
-
-    if current_user.update_attributes(strip_exif: privacy_params[:strip_exif])
-      flash[:notice] = t("users.update.settings_updated")
-    else
-      flash[:error] = t("users.update.settings_not_updated")
+    privacy_params = params.fetch(:user)
```

Comment:
We should still filter these params, otherwise the user can change stuff that he shouldn't be able to change.